quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -44,7 +51,13 @@
      */
     @ResponseBody
     @RequestMapping(value = "/api/v1/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public HttpStatus processJson(@RequestBody Quadruple quadruple) {
+    public HttpStatus processJson(@RequestBody Quadruple quadruple, @RequestParam(value = "collectionid", required = true) String collectionId) {
+        
+        Collection collection = collectionManager.findCollection(collectionId);
+        
+        if(Objects.isNull(collection)) {
```

Comment:
Is there any specific reason to use `Objects.isNull()?` Wouldn't simple null check work as well?

Reply:
Yes that would work as well, should I update it?

Reply:
I think simple check would make it uniform as it is more commonly used through out our apps